chore: fix regulation API doc for destinationIds (#1171)
rudderlabs/rudderstack-docs: docs/api/data-regulation-api.mdx

> - Specify **only `sourceIds`** when setting the `regulationType` to `suppress`. If no `sourceIds` are specified, RudderStack will suppress data from all the sources present in the workspace associated with the access token.

Reviewer: ```suggestion
- You must specify the **`sourceIds`** when the `regulationType` is set to `suppress`. If no `sourceIds` are specified, RudderStack will suppress data from all the sources present in the workspace associated with the access token.
```

Author: I believe, this is not required.

Reviewer: Just for the consistency purposes with the below :)

> - Specify **only `destinationIds`** when setting the `regulationType` to `suppress_with_delete`. 

Author: reference for the same can we found [here](https://github.com/rudderlabs/data-regulation-service/blob/78a0ea073148f318997dbe5684175ea0a25a496a/internal/api/user.go#L103-L107) in the code

> - Specify **only `destinationIds`** when setting the `regulationType` to `suppress_with_delete`. 

Reviewer: ```suggestion
- You must specify the **`destinationIds`** when the `regulationType` is set to `suppress_with_delete`. Otherwise, RudderStack throws an error.
```
